charts/teku/templates/_cmd.tpl

```diff
@@ -48,6 +49,7 @@
   --metrics-interface=0.0.0.0
   --metrics-host-allowlist=*
   --metrics-port={{ include "teku.metricsPort" . }}
+  --validators-proposer-default-fee-recipient="0x0000000000000000000000000000000000000000"
```

Comment:
Is this required? Will prysm fail to start without this? 

If this is required, then it would make sense to make the fee-recipient configurable via the values file. 
If it's not required, let's just remove this line from here.

Reply:
Teku requires it. Not prysm. 

Its needed for the lint checking only.

Reply:
Sorry, yeah I meant teku. What do you mean with lint checking? `make lint` ?

Reply:
https://github.com/skylenet/ethereum-helm-charts/runs/7904646730?check_suite_focus=true#step:9:4475

Reply:
See this comment, should be a good way to solve this instead of setting these flags directly on the chart: https://github.com/skylenet/ethereum-helm-charts/pull/96#issuecomment-1227188782